src/routes/placeFavorite.ts

```diff
@@ -0,0 +1,126 @@
+import * as placeFavoriteService from "@/services/placeFavorite";
+import { OpenAPIHono } from "@hono/zod-openapi";
+import authMiddleware from "@/middlewares/auth";
+import { Context } from "hono";
+import { bodySchema } from "@/schemas/placeFavorite";
+import { paramsSchema } from "@/schemas/placeFavorite";
+
+const placeFavoriteRoute = new OpenAPIHono();
+const API_TAGS = ["Favorite Places"];
+
+// Places Route
+placeFavoriteRoute.openapi(
+  {
+    method: "post",
+    path: "/",
+    summary: "Create favorite place",
+    tags: API_TAGS,
+    description: "Create favorite place.",
+    security: [{ AuthorizationBearer: [] }],
+    middleware: [authMiddleware],
+    request: {
+      body: {
+        content: {
+          "application/json": {
+            schema: bodySchema,
+          },
+        },
+      },
+    },
+    responses: {
+      201: {
+        description: "Success create favorite place",
+      },
+      400: {
+        description: "Failed to create favorite place",
+      },
+    },
+  },
+  async (c: Context) => {
+    const userId = c.get("userId") as string;
+    const body = await c.req.json();
```

Comment:
Since we have used zod at `bodySchema` , how about get body like this

> const body = c.req.valid("json");

Reply:
Okay, I will first discuss this with the tech lead, as it will change the validation request format on each route using the zValidator middleware

Reply:
ohh I see